cmd/wallet-web/src/main.js

```diff
@@ -90,13 +56,22 @@ new Vue({
     ...mapGetters('agent', { isAgentInitialized: 'isInitialized' }),
     ...mapGetters(['getAgentOpts']),
   },
-  i18n,
-  router,
-  render: (h) => h(App),
-}).$mount('#app');
+});
+
+app.config.globalProperties.$polyfill = polyfill;
+app.config.globalProperties.$webCredentialHandler = webCredentialHandler;
+app.use(router);
+app.use(store);
+app.use(i18n);
+app.component(
+  'ToastNotification',
+  require('./components/ToastNotification/ToastNotification').default
+);
 
 window.onbeforeunload = function () {
```

Comment:
just a follow up for you, you may have to check is this flush store is getting called when you navigate from page to page or when you refresh/close the page.